packages/components/src/color-picker/alpha.js

```diff
@@ -34,34 +34,64 @@ import { noop } from 'lodash';
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { Component, createRef } from '@wordpress/element';
+import { useState, useEffect, useRef } from '@wordpress/element';
 import { TAB } from '@wordpress/keycodes';
-import { pure } from '@wordpress/compose';
 
 /**
  * Internal dependencies
  */
 import { calculateAlphaChange } from './utils';
 import KeyboardShortcuts from '../keyboard-shortcuts';
 
-export class Alpha extends Component {
-	constructor() {
-		super( ...arguments );
+export default function Alpha( { rgb, hsl, onChange = noop } ) {
+	const [ mouseDown, setMouseDown ] = useState( false );
+	const container = useRef();
 
-		this.container = createRef();
-		this.increase = this.increase.bind( this );
-		this.decrease = this.decrease.bind( this );
-		this.handleChange = this.handleChange.bind( this );
-		this.handleMouseDown = this.handleMouseDown.bind( this );
-		this.handleMouseUp = this.handleMouseUp.bind( this );
-	}
+	const rgbString = `${ rgb.r },${ rgb.g },${ rgb.b }`;
+	const gradient = {
+		background: `linear-gradient(to right, rgba(${ rgbString }, 0) 0%, rgba(${ rgbString }, 1) 100%)`,
+	};
+	const pointerLocation = { left: `${ rgb.a * 100 }%` };
 
-	componentWillUnmount() {
-		this.unbindEventListeners();
-	}
+	const handleChange = ( e ) => {
```

Comment:
Personally, I prefer named functions, because they're easier to debug. I don't think there's a good reason to assign anonymous functions to a constant here.

Reply:
Thanks for reviewing, I've fixed that in daddou-ma/gutenberg@5fd8fc0